view/frontend/templates/bottomline.phtml

```diff
@@ -1,15 +1,6 @@
-<?php if($this->isRenderBottomlineQna()){ ?>
-<style>
-    .yotpo-icon-double-bubble, .yotpo-stars { margin: 0 5px 0 0 !important; }
-    div.yotpo.bottomLine { display: inline-block; margin: 0 10px 1px 0;}
-    .thumbnail .yotpo a { display: inline-block; }
-    div.yotpo.QABottomLine { display: inline-block; }
-    .product-info-main .product-reviews-summary{width:100%;}
-</style>
-<?php } ?>
-
+<div class="yotpoBottomLine">
```

Comment:
what it the purpose of this div? can we remove it?

Reply:
This div helps to keep bottomline and questions answers block in a row.

Reply:
This div helps to keep bottomline and questions answers block in a row.

Reply:
All other required changes have been done.